tcl/pkg.yaml

```diff
@@ -3,7 +3,7 @@ dependencies:
   - stage: base
 steps:
   - sources:
-      - url: https://downloads.sourceforge.net/project/tcl/Tcl/8.6.9/tcl-core8.6.9-src.tar.gz
+      - url: https://www.mirrorservice.org/sites/ftp.tcl.tk/pub/tcl/tcl8_6/tcl-core8.6.9-src.tar.gz
```

Comment:
sourceforge doesn't quite work from our Drone network, had to switch to another mirror